Game crashing

When I want to open librgetter gui my game is crashing everytime. Even when all other mods are disabled I cant even start the game cuz its insta crashing. 

Mods I've Used:
- AmbientSounds 
- AppleSkin 
- BBE 
- BadOptimizations 
- BaguetteLib 
- Blur+ 
- Boat Item View 
- Bobby 
- Borderless Mining Updated 
- Cloth Config v26.1 
- Cloud Save Manager 
- Clumps 
- Concurrent Chunk Management Engine 
- CreativeCore 
- Cull Leaves 
- Dynamic FPS 
- Entity Model Features 
- Entity Texture Features 
- EntityCulling 
- Euphoria Patcher 
- Fabric API 
- Fabric Language Kotlin 
- Fadeless 
- Falling Leaves 
- Fast IP Ping 
- Fast Noise 
- FerriteCore 
- Forge Config API Port 
- ImmediatelyFast 
- Iris 
- Jade 
- Just Enough Items 
- LambDynamicLights 
- Language Reload 
- LibrGetter 
- Litematica 
- Lithium 
- M.R.U 
- MaLiLib 
- MapDistanceFix 
- Mod Menu 
- ModernFix 
- More Culling 
- Mouse Tweaks 
- Mouse Wheelie 
- No Chat Reports 
- OptiGUI 
- Packet Fixer 
- Particular Reforged 
- Ping View 
- Placeholder API 
- Punchy 
- Reese's Sodium Options 
- Remove Reloading Screen 
- ScalableLux 
- ServerCore 
- Shulker Box Tooltip 
- Skyboxify 
- Smooth Scrolling 
- Smooth Swapping 
- Sodium 
- Sodium Extra 
- Sound Physics Remastered 
- Sounds 
- Tweakeroo 
- Version't 
- Very Many Players 
- Visuality 
- VoxelMap 
- YetAnotherConfigLib 
- e4mc 
- essential-container 
- kennytv's epic force close loading screen mod for Fabric 
- spark

Crash with mods:
```
java.lang.RuntimeException:` java.lang.NullPointerException: Cannot invoke "Object.getClass()" because "wrapper" is null
	at knot//dev.gxlg.versiont.api.R$RConstructor.newInst(R.java:571)
	at knot//dev.gxlg.versiont.gen.net.minecraft.client.gui.screens.inventory.BookViewScreen.lambda$new$0(BookViewScreen.java:42)
	at knot//dev.gxlg.versiont.api.types.Wrapper.<init>(Wrapper.java:17)
	at knot//dev.gxlg.versiont.gen.java.lang.Object.<init>(Object.java:19)
	at knot//dev.gxlg.versiont.gen.net.minecraft.client.gui.screens.Screen.<init>(Screen.java:19)
	at knot//dev.gxlg.versiont.gen.net.minecraft.client.gui.screens.inventory.BookViewScreen.<init>(BookViewScreen.java:46)
	at knot//dev.gxlg.versiont.gen.net.minecraft.client.gui.screens.inventory.BookViewScreen.<init>(BookViewScreen.java:42)
	at knot//dev.gxlg.librgetter.gui.ConfigScreen.<init>(ConfigScreen.java:12)
	at knot//dev.gxlg.librgetter.keybinds.ConfigMenuKeybind.execute(ConfigMenuKeybind.java:20)
	at knot//dev.gxlg.librgetter.keybinds.KeybindManager.lambda$register$1(KeybindManager.java:31)
	at knot//dev.gxlg.versiont.gen.net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents$EndTickI.lambda$static$1(ClientTickEvents$EndTickI.java:26)
	at knot//dev.gxlg.versiont.api.types.WrappedMethod.invoke(WrappedMethod.java:12)
	at knot//dev.gxlg.versiont.api.R.interceptInterface(R.java:188)
	at knot//dev.gxlg.versiont.api.R.lambda$interfaceInstance$0(R.java:205)
	at knot/jdk.proxy2/jdk.proxy2.$Proxy119.onEndTick(Unknown Source)
	at knot//net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents.lambda$static$3(ClientTickEvents.java:43)
	at knot//net.minecraft.client.Minecraft.handler$bjn000$fabric-lifecycle-events-v1$onEndTick(Minecraft.java:11179)
	at knot//net.minecraft.client.Minecraft.tick(Minecraft.java:2035)
	at knot//net.minecraft.client.Minecraft.runTick(Minecraft.java:1360)
	at knot//net.minecraft.client.Minecraft.run(Minecraft.java:991)
	at knot//net.minecraft.client.main.Main.main(Main.java:248)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:514)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:115)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: java.lang.NullPointerException: Cannot invoke "Object.getClass()" because "wrapper" is null
	at knot//dev.gxlg.versiont.api.R$Interceptor.intercept(R.java:326)
	at knot//dev.gxlg.librgetter.gui.ConfigScreenImpl.hashCode(Unknown Source)
	at java.base/java.util.WeakHashMap.hash(WeakHashMap.java:310)
	at java.base/java.util.WeakHashMap.get(WeakHashMap.java:409)
	at knot//dev.yumi.commons.event.FilteredEvent.forContext(FilteredEvent.java:185)
	at knot//dev.yumi.commons.event.FilteredEvent.forContext(FilteredEvent.java:164)
	at knot//net.minecraft.client.gui.screens.Screen.<init>(Screen.java:94)
	at knot//net.minecraft.client.gui.screens.Screen.<init>(Screen.java:91)
	at knot//net.minecraft.client.gui.screens.inventory.BookViewScreen.<init>(BookViewScreen.java:108)
	at knot//net.minecraft.client.gui.screens.inventory.BookViewScreen.<init>(BookViewScreen.java:100)
	at knot//dev.gxlg.librgetter.gui.ConfigScreenImpl.<init>(Unknown Source)
	at knot//dev.gxlg.versiont.api.R$RConstructor.newInst(R.java:569)
	... 26 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at knot//dev.gxlg.versiont.api.R$RConstructor.newInst(R.java:571)
	at knot//dev.gxlg.versiont.gen.net.minecraft.client.gui.screens.inventory.BookViewScreen.lambda$new$0(BookViewScreen.java:42)
	at knot//dev.gxlg.versiont.api.types.Wrapper.<init>(Wrapper.java:17)
	at knot//dev.gxlg.versiont.gen.java.lang.Object.<init>(Object.java:19)
	at knot//dev.gxlg.versiont.gen.net.minecraft.client.gui.screens.Screen.<init>(Screen.java:19)
	at knot//dev.gxlg.versiont.gen.net.minecraft.client.gui.screens.inventory.BookViewScreen.<init>(BookViewScreen.java:46)
	at knot//dev.gxlg.versiont.gen.net.minecraft.client.gui.screens.inventory.BookViewScreen.<init>(BookViewScreen.java:42)
	at knot//dev.gxlg.librgetter.gui.ConfigScreen.<init>(ConfigScreen.java:12)
	at knot//dev.gxlg.librgetter.keybinds.ConfigMenuKeybind.execute(ConfigMenuKeybind.java:20)
	at knot//dev.gxlg.librgetter.keybinds.KeybindManager.lambda$register$1(KeybindManager.java:31)
	at knot//dev.gxlg.versiont.gen.net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents$EndTickI.lambda$static$1(ClientTickEvents$EndTickI.java:26)
	at knot//dev.gxlg.versiont.api.types.WrappedMethod.invoke(WrappedMethod.java:12)
	at knot//dev.gxlg.versiont.api.R.interceptInterface(R.java:188)
	at knot//dev.gxlg.versiont.api.R.lambda$interfaceInstance$0(R.java:205)
	at knot/jdk.proxy2/jdk.proxy2.$Proxy119.onEndTick(Unknown Source)
	at knot//net.fabricmc.fabric.api.client.event.lifecycle.v1.ClientTickEvents.lambda$static$3(ClientTickEvents.java:43)
	at knot//net.minecraft.client.Minecraft.handler$bjn000$fabric-lifecycle-events-v1$onEndTick(Minecraft.java:11179)
```


Second Crash without any other mods (cant even start game):
```
java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'librgetter' at 'dev.gxlg.librgetter.Entrypoint'!
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$0(FabricLoaderImpl.java:409)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:407)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53)
	at knot//net.minecraft.client.Minecraft.<init>(Minecraft.java:477)
	at knot//net.minecraft.client.main.Main.main(Main.java:232)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:514)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:115)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: java.lang.NoClassDefFoundError: net/fabricmc/fabric/api/networking/v1/PacketSender
	at knot//dev.gxlg.librgetter.notifier.Notifier.<init>(Notifier.java:16)
	at knot//dev.gxlg.librgetter.services.loaders.NotifierLoader.init(NotifierLoader.java:14)
	at knot//dev.gxlg.librgetter.services.ServiceLoaderManager.initServiceLoader(ServiceLoaderManager.java:39)
	at knot//dev.gxlg.librgetter.services.ServiceLoaderManager.init(ServiceLoaderManager.java:27)
	at knot//dev.gxlg.librgetter.Entrypoint.onInitializeClient(Entrypoint.java:60)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:405)
	... 9 more
Caused by: java.lang.ClassNotFoundException: net.fabricmc.fabric.api.networking.v1.PacketSender
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:580)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:490)
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:239)
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:490)
	... 15 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.FabricLoaderImpl.lambda$invokeEntrypoints$0(FabricLoaderImpl.java:409)
	at net.fabricmc.loader.impl.util.ExceptionUtil.gatherExceptions(ExceptionUtil.java:33)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:407)
	at net.fabricmc.loader.impl.game.minecraft.Hooks.startClient(Hooks.java:53)
	at knot//net.minecraft.client.Minecraft.<init>(Minecraft.java:477)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Advanced Windows 32 Base API:10.0.26100.3624 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:User Experience Controls Library:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.26100.7623 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.26100.7705 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.26100.5074 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS Client API DLL:10.0.26100.1591 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.26100.8036 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP Helper API:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Windows NT BASE API Client DLL:10.0.26100.7920 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Windows NT BASE API Client DLL:10.0.26100.7920 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.26100.7920 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		OLEAUT32.dll:OLEAUT32.DLL:10.0.26100.7920 (WinBuild.160101.0800):Microsoft Corporation
		POWRPROF.dll:Power Profile Helper DLL:10.0.26100.4202 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Windows Performance Data Helper DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		RPCRT4.dll:Remote Procedure Call Runtime:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Windows Shell Common Dll:10.0.26100.7920 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll:User Mode Power Dependency Coordinator:10.0.26100.7019 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Multi-User Windows USER API Client DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.26100.7920 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.44.35208.0:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.26100.1150 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:Windows Socket 2.0 32-Bit DLL:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		aswAMSI.dll:Avast AMSI COM object:26.2.10802.0:Gen Digital Inc.
		aswhook.dll:Avast Hook Library:26.2.10802.0:Gen Digital Inc.
		bcrypt.dll:Windows Cryptographic Primitives Library:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.26100.7623 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM for Windows:10.0.26100.7705 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.26100.7920 (WinBuild.160101.0800):Microsoft Corporation
		fwpuclnt.dll:FWP/IPsec User-Mode API:10.0.26100.3915 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.26100.8036 (WinBuild.160101.0800):Microsoft Corporation
		java.dll:OpenJDK Platform binary:25.0.1.0:Microsoft
		javaw.exe:OpenJDK Platform binary:25.0.1.0:Microsoft
		jimage.dll:OpenJDK Platform binary:25.0.1.0:Microsoft
		jli.dll:OpenJDK Platform binary:25.0.1.0:Microsoft
		jna7849915573693727257.dll:JNA native library:7.0.2:Java(TM) Native Access (JNA)
		jvm.dll:OpenJDK 64-Bit server VM:25.0.1.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.26100.7920 (WinBuild.160101.0800):Microsoft Corporation
		management.dll:OpenJDK Platform binary:25.0.1.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:25.0.1.0:Microsoft
		msvcp140.dll:Microsoft® C Runtime Library:14.44.35208.0:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.26100.7623 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.26100.7623 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Microsoft Windows Sockets 2.0 Service Provider:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:E-mail Naming Shim Provider:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Windows NCrypt Router:10.0.26100.1591 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:25.0.1.0:Microsoft
		nio.dll:OpenJDK Platform binary:25.0.1.0:Microsoft
		nlansp_c.dll:NLA Namespace Service Provider DLL:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		ntdll.dll:NT Layer DLL:10.0.26100.7920 (WinBuild.160101.0800):Microsoft Corporation
		ole32.dll:Microsoft OLE for Windows:10.0.26100.6725 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Windows System Performance Objects DLL:10.0.26100.7705 (WinBuild.160101.0800):Microsoft Corporation
		pfclient.dll:SysMain Client:10.0.26100.4202 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.26100.7309 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.26100.7920 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Shell Light-weight Utility Library:10.0.26100.1 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:25.0.1.0:Microsoft
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.26100.7623 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.44.35208.0:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:25.0.1.0:Microsoft
		win32u.dll:Win32u:10.0.26100.8036 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage API:10.0.26100.7840 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.26100.1882 (WinBuild.160101.0800):Microsoft Corporation
		wintypes.dll:Windows Base Types DLL:10.0.26100.7705 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.26100.5074 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:25.0.1.0:Microsoft
Stacktrace:
	at knot//net.minecraft.client.main.Main.main(Main.java:232)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:514)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:72)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:115)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:129)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
```

--- a/build.gradle
+++ b/build.gradle
@@ -1,6 +1,6 @@
 plugins {
     id "net.fabricmc.fabric-loom" version "${loom_version}"
-    id "dev.gxlg.versiont-toolchain" version "1.0.2"
+    id "dev.gxlg.versiont-toolchain" version "1.1.0"
 }
 
 version = project.mod_version

--- a/gradle.properties
+++ b/gradle.properties
@@ -11,9 +11,9 @@ maven_group=dev.gxlg
 archives_base_name=librgetter
 
 # Fabric Properties
-minecraft_version=26.1
-loader_version=0.18.5
+minecraft_version=26.1.1
+loader_version=0.18.6
 loom_version=1.15-SNAPSHOT
 
 # Fabric API
-fabric_api_version=0.144.3+26.1
+fabric_api_version=0.145.4+26.1.1

--- a/src/main/java/dev/gxlg/librgetter/utils/types/TradeOfferData.java
+++ b/src/main/java/dev/gxlg/librgetter/utils/types/TradeOfferData.java
@@ -1,13 +1,15 @@
 package dev.gxlg.librgetter.utils.types;
 
-import dev.gxlg.versiont.gen.net.minecraft.world.item.trading.MerchantOffers;
+import dev.gxlg.versiont.gen.net.minecraft.world.item.trading.MerchantOffer;
+
+import java.util.List;
 
 public class TradeOfferData {
     private final boolean canRefresh;
 
-    private final MerchantOffers tradeOfferList;
+    private final List<MerchantOffer> tradeOfferList;
 
-    private TradeOfferData(boolean canRefresh, MerchantOffers list) {
+    private TradeOfferData(boolean canRefresh, List<MerchantOffer> list) {
         this.canRefresh = canRefresh;
         tradeOfferList = list;
     }
@@ -16,7 +18,7 @@ public class TradeOfferData {
         return canRefresh;
     }
 
-    public MerchantOffers getTradeOfferList() {
+    public List<MerchantOffer> getTradeOfferList() {
         return tradeOfferList;
     }
 
@@ -24,7 +26,7 @@ public class TradeOfferData {
         return new TradeOfferData(false, null);
     }
 
-    public static TradeOfferData offers(MerchantOffers tradeOfferList) {
+    public static TradeOfferData offers(List<MerchantOffer> tradeOfferList) {
         return new TradeOfferData(true, tradeOfferList);
     }
 }

--- a/src/main/java/dev/gxlg/librgetter/utils/types/messages/Message.java
+++ b/src/main/java/dev/gxlg/librgetter/utils/types/messages/Message.java
@@ -2,10 +2,11 @@ package dev.gxlg.librgetter.utils.types.messages;
 
 import dev.gxlg.versiont.gen.net.minecraft.ChatFormatting;
 import dev.gxlg.versiont.gen.net.minecraft.network.chat.MutableComponent;
+import dev.gxlg.versiont.gen.net.minecraft.network.chat.Style;
 
 public abstract class Message {
     public final MutableComponent getComponent() {
-        return buildComponent().withStyle(getColor());
+        return buildComponent().withStyle(Style.EMPTY().withColor(getColor()));
     }
 
     protected abstract MutableComponent buildComponent();

--- a/src/main/java/dev/gxlg/librgetter/worker/tasks/FinalizeSearchTask.java
+++ b/src/main/java/dev/gxlg/librgetter/worker/tasks/FinalizeSearchTask.java
@@ -14,12 +14,13 @@ import dev.gxlg.librgetter.worker.types.task.Task;
 import dev.gxlg.versiont.gen.net.minecraft.world.entity.player.Inventory;
 import dev.gxlg.versiont.gen.net.minecraft.world.item.ItemStack;
 import dev.gxlg.versiont.gen.net.minecraft.world.item.trading.MerchantOffer;
-import dev.gxlg.versiont.gen.net.minecraft.world.item.trading.MerchantOffers;
+
+import java.util.List;
 
 public class FinalizeSearchTask extends Task {
-    private final MerchantOffers offers;
+    private final List<MerchantOffer> offers;
 
-    public FinalizeSearchTask(MerchantOffers offers) {
+    public FinalizeSearchTask(List<MerchantOffer> offers) {
         this.offers = offers;
     }
 
@@ -38,9 +39,9 @@ public class FinalizeSearchTask extends Task {
         MinecraftData minecraftData = taskContext.minecraftData();
         Inventory inventory = minecraftData.localPlayer.getInventory();
         int buy;
-        if (canBuy(inventory, (MerchantOffer) offers.get(0))) {
+        if (canBuy(inventory, offers.get(0))) {
             buy = 0;
-        } else if (canBuy(inventory, (MerchantOffer) offers.get(1))) {
+        } else if (canBuy(inventory, offers.get(1))) {
             buy = 1;
         } else {
             throw new CanNotLockException();

--- a/src/main/java/dev/gxlg/librgetter/worker/tasks/ParseAndMatchTradesTask.java
+++ b/src/main/java/dev/gxlg/librgetter/worker/tasks/ParseAndMatchTradesTask.java
@@ -20,16 +20,15 @@ import dev.gxlg.librgetter.worker.types.switcher.TaskSwitch;
 import dev.gxlg.librgetter.worker.types.task.Task;
 import dev.gxlg.versiont.gen.net.minecraft.world.item.Items;
 import dev.gxlg.versiont.gen.net.minecraft.world.item.trading.MerchantOffer;
-import dev.gxlg.versiont.gen.net.minecraft.world.item.trading.MerchantOffers;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
 
 public class ParseAndMatchTradesTask extends Task {
-    private final MerchantOffers offers;
+    private final List<MerchantOffer> offers;
 
-    public ParseAndMatchTradesTask(MerchantOffers trades) {
+    public ParseAndMatchTradesTask(List<MerchantOffer> trades) {
         this.offers = trades;
     }
 
@@ -40,7 +39,7 @@ public class ParseAndMatchTradesTask extends Task {
             if (i >= 2 && configManager.getOptions(Config.MATCH_MODE) == MatchMode.VANILLA) {
                 break;
             }
-            MerchantOffer offer = (MerchantOffer) offers.get(i);
+            MerchantOffer offer = offers.get(i);
             if (!isEnchantmentTrade(offer)) {
                 continue;
             }

--- a/src/main/resources/fabric.mod.json
+++ b/src/main/resources/fabric.mod.json
@@ -25,7 +25,7 @@
   ],
   "depends": {
     "minecraft": ">=1.17",
-    "versiont": "1.0.2"
+    "versiont": "1.1.0"
   },
   "conflicts": {
     "offershud": "<1.9.0"

--- a/src/main/resources/versiont.mapping
+++ b/src/main/resources/versiont.mapping
@@ -22,12 +22,6 @@ adapter java.util.Set -> dev.gxlg.librgetter.utils.adapters.SetAdapter
 
 import java.util.concurrent.CompletableFuture
 
-import !java.util.ArrayList
-
-class ArrayList
-    [Object] get(int i)
-    int size()
-
 # ------------------------------------------------------------------------------------------------------------- #
 # Core
 
@@ -195,8 +189,6 @@ class Identifier
     String method_12836/getNamespace()
     String method_12832/getPath()
     String toString()
-
-class MerchantOffers extends ArrayList
 
 # ------------------------------------------------------------------------------------------------------------- #
 # Geometry
@@ -659,7 +651,7 @@ import net.minecraft.class_8673/net.minecraft.client.multiplayer.ClientCommonPac
 
 class ClientboundMerchantOffersPacket extends Packet
     int method_19459/getVillagerXp()
-    MerchantOffers method_17590/getOffers()
+    List<MerchantOffer> method_17590/getOffers()
 
 class ClientboundOpenScreenPacket extends Packet
     MenuType method_17593/getType()

--- a/src/main/resources/versiont.mapping
+++ b/src/main/resources/versiont.mapping
@@ -104,7 +104,6 @@ import net.minecraft.class_2378/net.minecraft.core.Registry
 import net.minecraft.class_7923/net.minecraft.core.registries.BuiltInRegistries
 import net.minecraft.class_5455/net.minecraft.core.RegistryAccess
 import net.minecraft.class_6885$class_6888/net.minecraft.core.HolderSet$Named
-import net.minecraft.class_1916/net.minecraft.world.item.trading.MerchantOffers
 
 class Holder
     boolean method_40225/is(ResourceKey resourceKey)

